docs/setup/tomcat.md

```diff
@@ -520,8 +520,6 @@ JAVA_OPTS="$JAVA_OPTS \
 
 JAVA_OPTS="$JAVA_OPTS \
             -server \
-            -XX:+UseParNewGC \
-            -XX:ParallelGCThreads=2 \
             -XX:SoftRefLRUPolicyMSPerMB=36000 \
             -XX:+UseConcMarkSweepGC"
```

Comment:
ConcMarkSweepGC not relevant anymore, now that G1 GC is available by default (see https://docs.geoserver.org/latest/en/user/production/container.html)